src/App.php

```diff
@@ -49,6 +49,18 @@
 class App extends AbstractApp {
 
 	/**
+	 * @var array Languages supported by the app
+	 */
+	public $supportedLanguages = [ 'en', 'fr' ];
```

Comment:
May I ask what is the purpose of having a public property and a getter at the same time. Thank you.